drivers/adc/ad4080/ad4080.c

```diff
@@ -879,6 +879,190 @@ int ad4080_get_fifo_mode(struct ad4080_dev *dev,
 	return 0;
 }
 
+/**
+ * @brief Set AD4080 FIFO Watermark
+ * @param dev - The device structure.
+ * @param fifo_watermark - The FIFO Watermark.
+ * @return 0 in case of success, negative error code otherwise
+ */
+int ad4080_set_fifo_watermark(struct ad4080_dev *dev,
+			      uint16_t fifo_watermark)
+{
+	int ret;
+
+	if (!dev || fifo_watermark > AD4080_FIFO_SIZE)
+		return -EINVAL;
+
+	ret = ad4080_write(dev,
+			   AD4080_REG_FIFO_WATERMARK,
+			   no_os_field_get(BYTE_ADDR_L, fifo_watermark));
+	if (ret) {
```

Comment:
the convention is the same as for linux drivers. If only one statement within a conditional expression, then no brackets needed.

Reply:
Fixed